FFCDemoPaymentService/Migrations/20200622145749_InitialCreate.cs

```diff
@@ -1,12 +1,10 @@
-﻿using Microsoft.EntityFrameworkCore.Migrations;
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
 using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
-using System.Diagnostics.CodeAnalysis;
-
 
 namespace FFCDemoPaymentService.Migrations
 {
-    [ExcludeFromCodeCoverage]
```

Comment:
I'm not a fan of including auto generated code first migrations as part of code coverage because 99% of time developers don't make any changes and to cover them with tests would be testing code you didn't write.

Over time as the service is iterated with more and more database changes then the majority of the code can very quickly become migrations meaning the code coverage percentage loses value, particularly with microservices.

I'd suggest using the `ExcludeFromCodeCoverage` flag on these files.